src/index.js

```diff
@@ -202,11 +202,16 @@ AFRAME.registerSystem('input-mapping', {
 
   keyboardHandler: function (event) {
     this.checkValidInputMapping();
-    if (mappings && mappings.keyboard) {
-      mappings = mappings.keyboard;
-      var key = event.keyCode === 32 ? 'Space' : event.key;
-      var keyEvent = (key + '_' + event.type.substr(3)).toLowerCase();
-      var mapEvent = mappings[keyEvent];
+    if (mappings &&
+        mappings.mappings &&
```

Comment:
`mappings` is the top-level config:
`{behaviours: [], mappings: {task1 : {...}, task2 : {...} }`
`mappings.mappings` is the set of action-sets.
`AFRAME.currentInputMapping` is the name of one of these action sets (or `null`, but we `checkValidInputMapping` before this line so we'll get a warning in this case.)